main.go

```diff
@@ -306,17 +304,20 @@ func getKeyStore(datastore string, keyName string) *Keystore {
 	return NewKeystore(dirstore, keyName)
 }
 
-func loadTenantToken(datastore string) ([]byte, error) {
+func loadTenantToken(config *menderConfig, datastore string) ([]byte, error) {
+
 	dirstore := NewDirStore(datastore)
-	raw, err := dirstore.ReadAll(defaultTenantTokenFile)
+
+	raw, err := dirstore.ReadAll(config.GetTenantToken())
```

Comment:
This might be slightly confusing. We're letting the user configure the name of tenant token file, but it must be located inside mender data store directory. Why give the user such ability even though the file must be inside a very specific directory?